_data/tools.yml

```diff
@@ -88,6 +88,21 @@
       url: https://www.datadoghq.com/
     - urltext: Documentation for Trino integration
       url: https://docs.datadoghq.com/integrations/trino/
+- name: DBeaver
+  anchor: DBeaver
+  category: client
+  logo: /assets/images/logos/dbeaver.png
+  logosmall: /assets/images/logos/dbeaver-small.png
+  description: |
+      DBeaver Community is a free cross-platform database tool for developers,
+      database administrators, analysts, and everyone working with data. It
+      supports all popular SQL databases like MySQL, MariaDB, PostgreSQL,
+      SQLite, Apache Family, and more.
+  links:
+    - urltext: DBeaver
+      url: https://dbeaver.io/
+    - urltext: DBeaver documentation
+      url: https://github.com/dbeaver/dbeaver/wiki
```

Comment:
Is there a page about Trino?

Reply:
Also add another link .. to https://docs.starburst.io/data-consumer/clients/dbeaver.html